src/go/install.sh

```diff
@@ -191,15 +191,18 @@ else
 fi
 
 # Install Go tools that are isImportant && !replacedByGopls based on
-# https://github.com/golang/vscode-go/blob/v0.31.1/src/goToolsInformation.ts
+# https://github.com/golang/vscode-go/blob/v0.38.0/src/goToolsInformation.ts
```

Comment:
@bamurtaugh/ @Chuxel any concerns with syncing the Go tools with latest https://github.com/golang/vscode-go/blob/v0.38.0/src/goToolsInformation.ts ?

Reply:
No concerns from my end - that's really the source of truth.